### PR TITLE
Engineer Workboots

### PR DESCRIPTION
**Pull request by DaveTheHeadcrab**
**DISCLAIMER:** _THIS PR IS PURELY COSMETIC. THESE BOOTS WILL NOT MAKE YOU LESS SHIT AT ENGINEERING. STOP BEING SHIT AT ENGINEERING._

Adds workboots for engineers to replace the current orange sneakers they start with.

Pics:
workboots
bootsexample

:cl: DaveTheHeadcrab
rscadd: Adds workboots to be worn by engineers.
/:cl:

### CODE REVIEW

**Comment by Deanthelis** [comment]
BUT DO THEY MAKE NOISE THOUGH!?

**Comment by tigercat2000** [comment]
not sure i prefer these over the generic orange shoes
probably should have both in the wardrobe :tiger:

**Comment by DaveTheHeadcrab** [comment]
DEAN DO YOU KNOW WHAT THE FUCKING WORD **COSMETIC** MEANS BECAUSE CURRENTLY I DO BELIEVE YOU ARE ILLITERATE AND/OR MENTALLY DISABLED.

**Comment by FalseIncarnate** [comment]
inb4 traitor variant that is steel-toed and grants massive unarmed attack damage and stun via groin-kicking.

Also, give these to atmos techs?

**Comment by Regen1** [comment]
NO THESE ARE WAY TOO POWERFUL FUCKING POWERCREEP STOP BUFFING THAT ONE DEPARTMENT GOODAMNIT

**Comment by DaveTheHeadcrab** [comment]
I gave these to everyone who started with orange boots. I guess atmos techs can have them too since false asked super nicely.

**Comment by Deanthelis** [comment]
IT'S A COSMETIC NOISE OKAY DAVE
MAKE THEM SQUEAK WHEN WET LIKE REAL BOOTS DAVE

**Comment by DaveTheHeadcrab** [comment]
Have I appeased you, @FalseIncarnate ?

**Comment by Deanthelis** [comment]
WATER SQUEAKS ARE MANDATORY

**Comment by FalseIncarnate** [comment]
No.
:+1:
But it's a start.

**Comment by Spacemanspark** [comment]
:+1:

**Comment by Norgad** [comment]
Is there a vox sprite? (With the digitigrade feet shape)

**Comment by MindLeaker** [comment]
This seems a bit too OP I'm afraid 

:-1: from me. Give all who wear it reduced stun times every second tick and I might change my mind.

**Comment by Citinited** [comment]
I actually love -FalseIncarnate's idea of a traitorous version of these.

:+1: to these too.

**Comment by Dumbdumn5** [comment]
I REFUSE TO LET YOU ADD THESE, JESUS CHRIST, MY EYES ARE GOING TO BLEED. LOOK HOW UNBALANCED THEY ARE! MOTHER OF FUCK, THINK OF THE CHILDREN! DAVE, THIS PR MAKES ENGINEERING INTO A STATION OF ITS OWN THAT NO ANTAG CAN EVER TOUCH, YOU MIGHT AS WELL GIVE THEM GOD MODE, HOLY SHIT! 

(It's better than the prisoner shoes they start with, and it looks kind of nice. :+1:)

**Comment by Fox-McCloud** [comment]
I think these are ugly as hell, but everyone seems to like them.

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 1]
```diff
@@ -99,6 +99,12 @@
 	item_color = "jacksandal"
 	species_restricted = null
 
+/obj/item/clothing/shoes/workboots
+	name = "work boots"
+	desc = "Thick-soled boots for industrial work environments."
+	can_cut_open = 1
+	icon_state = "workboots"
```
Comment (Fox-McCloud) [inline code/modules/clothing/shoes/miscellaneous.dm]: gives these

`item_state = "jackboots"`
and a strip+put on delay of 40

**Comment by Faryaa** [comment]
I like how they look. Also please give them to mechanic and miners. Maybe add them to clothing machine as well.

**Comment by tigercat2000** [comment]
you should use the loadout system for these iyam
they look like shit to some of us

**Comment by Faryaa** [comment]
Loadout system would be neat but we could just add boots inside miners and engineering clothes lockers. So people could wear them if they really want to. Or stick to their old shoes if they want.

**Comment by DaveTheHeadcrab** [comment]
Mechnic already gets them.
And if they look like shit there's a convenient place you can go with an absurd number of clothing vendors to cater to whatever style you so desire.

**Comment by CrAzYPiLoT-SS13** [comment]
I like these a lot better than the shoes the engineers get currently.

**Comment by TheDZD** [comment]
I like the item_state for these, the icon_state looks horribly thought in my opinion. Overall :+1: though.

**Comment by Spacemanspark** [comment]
:+1: to all, love these boots

**Comment by Fox-McCloud** [comment]
>I like the item_state for these

Actually, the on mob sprite goes off of the icon state...item state is just for in-hands.

Go figure.

**Comment by Davdichan** [comment]
Miners should definitely get these too. Adding extra pairs to uniform closets would be a good idea for job transfers. Clothing vendor too

Steel toes that give feet armor would be amusing but pointless.

**Pull request merged**